files/build_templates/sonic_debian_extension.j2

```diff
@@ -770,6 +770,15 @@ if [ "${PIPESTATUS[0]}" != "0" ]; then
     exit 1
 fi
 
+sudo mkdir -p $FILESYSTEM_ROOT/$SONIC_PACKAGE_MANAGER_FOLDER/manifests
+target_machine="$TARGET_MACHINE" j2 $BUILD_TEMPLATES/default_manifest.j2 | sudo tee $FILESYSTEM_ROOT/$SONIC_PACKAGE_MANAGER_FOLDER/manifests/default_manifest
```

Comment:
Where is ```target_machine``` consumed in default_manifest.j2?

Reply:
target_machine is not required in default_manifest.j2 
Hence removed it now.